doc/integrator/structure.rst

```diff
@@ -0,0 +1,58 @@
+.. _integrator_structure:
+
+Project structure
+=================
+
+We can split the project in 3 parts.
+
+Configuration
+-------------
+
+Thoes files are use to build the config image.
+
+``mapserver``: all the files needed by Mapserver, will be in the folder ``/etc/mapserver`` in the config
+image.
+
+``qgisserver``: all the files needed by QGIS server, will be in the folder ``/etc/qgisserver`` in the config
+image.
+
+``tilegeneration``: all the files needed by TileCloud-chain, will be in the folder ``/etc/tilegeneration``
+in the config image.
+
+``print/print-apps``: all the files needed my Mapfish print, will be in the folder
+``/usr/local/tomcat/webapps/ROOT/print-apps`` in the config image.
+
+``geoportal/vars*.yaml``, ``geoportal/CONST_vars.yaml``, ``geoportal/CONST_config-schema.yaml``: All the
+file needed to build the config, will be in the file ``/etc/geomapfish/config.yaml`` in the config image.
+
+``geoportal/geomapfish_geoportal/static``: All the static file needed for your application, will be
+available ins the static vies with cache buster. Will be in the folder ``/etc/geomapfish/static`` in the
+config image.
+
+``geoportal/geomapfish_geoportal/locale``: Call the files needed to localise your application, will be in
+the folder ``/etc/geomapfish/locale`` in the config image.
+
+Custom application
+------------------
+
+.. note::
+
+    This part didn't exists in the simple application mode
+
+``geoportal/geomapfish_geoportal`` the pyramid application.
+
+``geoportal/webpack.*`` the webpack build configuration.
+
+other files in ``geoportal`` exxantially files rlated to the build.
```

Comment:
exxantially files rlated to the build.
probably:
"essentially files related to the build" ?